daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/Runner.scala

```diff
@@ -438,6 +438,7 @@ private[lf] class Runner(
       Speedy.Machine.fromPureSExpr(
         extendedCompiledPackages,
         script.expr,
+        iterationsBetweenInterruptions = 100000,
```

Comment:
Here we were using a wrong default (`Max.Long`). This is catch by the additional test `scriptMachine`

Reply:
would it make sense to change the default in the definition of `fromPureSExr`?

Reply:
it is problably fine.